### DISCUSSION TITLE
CSP issue with inline-styles

### DISCUSSION THREAD

**Discussion by samods2** [rappasoft/laravel-livewire-tables#1783] [Q&A] [answered]
Hello,

When using strict CSP policy, there are lot of errors thrown by the package mentioning about the inline-styles. Is there any way to by pass this one?

**Comment by lrljoe** [comment]
Are you using Bootstrap or Tailwind?

**Reply by lrljoe** [reply]
I'm assuming Bootstrap here, I could migrate the styles (other than the numberrange filter) into classes

**Reply by lrljoe** [reply, marked as answer]
#1788  - should address some of this!
